src/riot-rs-embassy/src/lib.rs

```diff
@@ -51,15 +51,26 @@ use riot_rs_debug::log::debug;
 pub use linkme::{self, distributed_slice};
 pub use static_cell::{ConstStaticCell, StaticCell};
 
-// Used by a macro we provide
-pub use embassy_executor;
-pub use embassy_executor::Spawner;
+// This module is whole-sale re-exported at `riot_rs::`.
+pub mod api {
```

Comment:
So before we did `pub use riot_rs_embassy::*` in `riot-rs`. IMO that needs more consideration. At least, if there's a `crates` export in embassy *and* in riot-rs (where the latter is a supersed of the former), due to name clash we can't wildcard import. As a crutch, I added this api module that collects everthing that riot-rs should re-export (excluding the crates which are in the module below).